recipes/mayavi/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "4.5.0" %}
+
+package:
+  name: mayavi
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/enthought/mayavi.git
+  git_rev: {{ version }}
+
+build:
+  entry_points:
+    - mayavi2 = mayavi.scripts.mayavi2:main
+    - tvtk_doc = tvtk.tools.tvtk_doc:main
+  skip: True  # [osx or (linux and py34) or win]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - numpy
+    - setuptools
+    - vtk
+    - traitsui
+    - apptools
+  run:
+    - python
+    - numpy
+    - vtk  ==7.1.0
+    - traitsui
+    - apptools
+    - envisage
+    - pyqt
+    - setuptools
```

Comment:
Does `mayavi` needs `pkg_resources`? If not we can remove `setuptools` from the run dependencies.